src/libexpr-c/nix_api_expr.cc

```diff
@@ -14,10 +12,37 @@
 #include "nix_api_util.h"
 #include "nix_api_util_internal.h"
 
+#include <cstring>
+#include <stdexcept>
+#include <string>
```

Comment:
I think putting the external ones at the top is better? It's the rest of our code that should be changed to do this.

Reply:
:facepalm:
Done.